### PR TITLE
[IcM] Cherry-Pick Changes from #1357 to release/3.4.4

### PR DESCRIPTION
**Pull request by iambmelt**
Brings changes from
#1357

This issue was originally committed for June in the following IcM:
https://portal.microsofticm.com/imp/v3/incidents/details/239658713/home

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,7 @@
+Version 3.4.4
+----------
+- [PATCH] Disregard pageload errors for the non-primary frame during interactive auth (#1357)
```
Comment (shoatman) [inline changelog.txt:3]: Based on the android documentation all errors (images, scripts, iframes) that fail to load cause a page load error.

File: common/src/main/java/com/microsoft/identity/common/internal/ui/webview/OAuth2WebViewClient.java [hunk 2]
```diff
@@ -145,7 +145,15 @@ public void onReceivedError(final WebView view,
     public void onReceivedError(@NonNull final WebView view,
```
Comment (shoatman) [inline common/src/main/java/com/microsoft/identity/common/internal/ui/webview/OAuth2WebViewClient.java:145]: I think it's worth adding some javadoc here to explain what's going on... you may want to link to the android documentation as well.

**Review by shoatman — APPROVED** [review]
Suggested some additional javadoc, but otherwise good to go.  Thanks!

File: common/src/main/java/com/microsoft/identity/common/internal/ui/webview/OAuth2WebViewClient.java (hunk 2 above)
Comment (iambmelt) [reply common/src/main/java/com/microsoft/identity/common/internal/ui/webview/OAuth2WebViewClient.java:145]: ACK

Comment (iambmelt) [reply common/src/main/java/com/microsoft/identity/common/internal/ui/webview/OAuth2WebViewClient.java:160]: 77225978b2c3236b3a18051f4de76f040fdfeb4e

**Review by iamgusain — APPROVED** [review]

**Pull request merged**